test/core/WebGLRenderer.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+function isWebGLSupported(fn)
+{
+    return PIXI.utils.isWebGLSupported() ? fn : function () {}; // eslint-disable-line no-empty-function
```

Comment:
This will still output different tests for each environment. I would have this return `undefined` instead of an empty function. Then use it on the `it()` blocks.

That way each `it()` test registers, and it will output "skipped" instead of not appearing at all.